sale_order_product_recommendation/views/sale_order_view.xml

```diff
@@ -1,22 +1,18 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<!-- Copyright 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
-     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 <odoo>
-
     <record id="view_order_form" model="ir.ui.view">
         <field name="name">Recommended products button</field>
         <field name="model">sale.order</field>
         <field name="inherit_id" ref="sale.view_order_form" />
         <field name="arch" type="xml">
-            <header>
+            <xpath expr="//form//header//button[@name='action_draft']" position="after">
```

Comment:
Why changing this positioning? I'm not sure if the button is going to be shown when needed.

Reply:
Is the button always shown?

Reply:
Please confirm which position this button should be left in.